### ISSUE TITLE
Installation on a webserver (Apache)

### ISSUE THREAD

**Issue by haGibear** [byro/byro#272]
This might sound stupid, but is there a way to install byro on a webserver without terminal access?

**Comment by rixx** [comment]
No, you have to have access to your server to install byro. There are no hosted or bundled offers at the moment.
